examples/simple_reverse.coffee

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env coffee
```

Comment:
You don't need this unless the file is meant to be executable (which it probably shouldn't be).